src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,128 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connecting a libSQL database for production
+
+Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server
```

Comment:
It isn't needed for local DBs (files and in-memory).
```suggestion
- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server, if needed
```

Maybe it could be more detailed, I'm not sure.

Reply:
I agree that it could be more detailed, like "this isn't needed for local DBs"